homeassistant/components/hunterdouglas_powerview/config_flow.py

```diff
@@ -18,9 +19,9 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
 HAP_SUFFIX = "._hap._tcp.local."
-POWERVIEW_SUFFIX = "._powerview._tcp.local."
+POWERVIEW_G2_SUFFIX = "._powerview._tcp.local."
+POWERVIEW_G3_SUFFIX = ".powerview-g3.local."
```

Comment:
```suggestion
POWERVIEW_G3_SUFFIX = ".powerview-g3._tcp.local."
```

Reply:
As above

Reply:
I won't be near my computer for a couple of days - will this one need a hassfest update too or just a commit here ?

Reply:
It shouldn't need a hassfest update as its mostly manifest.json changes that trigger the need to run that